app/config/config.yml

```diff
@@ -61,6 +61,7 @@ twig:
     debug: '%kernel.debug%'
     strict_variables: '%kernel.debug%'
     form_themes: ['bootstrap_3_horizontal_layout.html.twig']
+    cache:  false
```

Comment:
Why ???

Reply:
local change that should not have been in the PR.